compiler/transformation/transform.rs

```diff
@@ -61,3 +62,53 @@ fn prune_redundant_roleplayer_deduplication(conjunction: &mut Conjunction, block
         }
     });
 }
+
+pub fn optimize_away_statically_unsatisfiable_conjunctions(
```

Comment:
Our current architecture has each transformation in an independent file, so let's give this one its own called, maybe `unsatisfiability.rs`?

Reply:
I broke that pattern two weeks ago. We're going to have a bunch of files containing a 20 line function this way.